gpmora10/Session04/dictmailroom.py

```diff
@@ -0,0 +1,88 @@
+donors = [
+            {"Name":"Donor", "Amount":"Total Donated", "Number":"Number of Donations"},
+            {"Name":"Gregory Blazek", "Amount":30, "Number":1},
+            {"Name":"Tiffany Huntley", "Amount":1070, "Number":2},
+            {"Name":"Gabriel Mora", "Amount":2050, "Number":3},
+            {"Name":"Courtney Barrett", "Amount":765, "Number":2},
+            {"Name":"Andrew Noodle", "Amount":2098, "Number":3}
+]
+individualDonations = [
+    {"Name":"Gregory Blazek","Amount":30},
+    {"Name":"Tiffany Huntley","Amount":1000},
+    {"Name":"Tiffany Huntley","Amount":70},
+    {"Name":"Gabriel Mora","Amount":1000},
+    {"Name":"Gabriel Mora","Amount":1000},
+    {"Name":"Gabriel Mora","Amount":50},
+    {"Name":"Courtney Barrett","Amount":750},
+    {"Name":"Courtney Barrett","Amount":15},
+    {"Name":"Andrew Noodle","Amount":1000},
+    {"Name":"Andrew Noodle","Amount":1000},
+    {"Name":"Andrew Noodle","Amount":98}
+
+]
+
+def showMenu():
+    usrchoice = input("Please choose an option:\n"
+          "1. Send a Thank You\n"
+          "2. Create a Report\n"
+          "3. quit\n"
+          "Type in the option or the number: ")
+    return usrchoice.lower()
+
+def sendThanks():
+
+    thanksChoice = input("Enter a donor's name or enter \"list\" to see a list of donors: ")
```

Comment:
You don't need to use escaping for quotes if you rewrite the line as such:
```
thanksChoice = input('Enter a donor's name or enter "list" to see a list of donors: ')
```
You can use alternating quotes where ever you like.  There is also the use of `"""` quotes which I think would be worth your while to look up. :)